### DISCUSSION TITLE
a question about cmdPosition

### DISCUSSION THREAD

**Discussion by SSR017** [USC-ACTLab/crazyswarm#794] [Q&A]
I used the **cmdposition** command to control the drone after **takeoff** using **takeoff**, but the plane did not execute as expected and flew slowly towards the front. Do you have any opinions on this? Is it a problem that **cmdposition** cannot be mixed with other control methods

**Comment by jhzhang2077** [comment]
It looks like wrong configuration of the CFs. Please check the direction and markers.
